app/controllers/verify/finance_controller.rb

```diff
@@ -28,15 +26,31 @@ def create
 
     private
 
+    def step_name
+      :financials
+    end
+
+    def confirm_step_needed
+      redirect_to verify_address_path if idv_session.financials_confirmation == true
+    end
+
+    def view_model(error: nil)
```

Comment:
looks like `view_model` is never called with an `error:`, can we drop the argument?